terraform/cluster/do/outputs.tf

```diff
@@ -1,24 +1,24 @@
 output "ca" {
-  depends_on = [digitalocean_kubernetes_cluster.rack]
+  depends_on = [ null_resource.wait_k8s_cluster ]
```

Comment:
Might be wrong but I think the only one that will need a null resource is the AWS install because the output value is from a different resource than it had on the "depends_on" clause.
All others seem to use the same resource as the depends on clause - just removing depends_on will create an implicit dependency.

Reply:
I haven't tested it on DO/Azure/GCP, it's working on AWS.